core/templates/domain/opportunity/ExplorationOpportunitySummaryObjectFactory.ts

```diff
@@ -24,6 +24,15 @@ export interface ITranslationCountsDict {
   [languageCode: string]: number
 }
 
+export interface IExplorationOpportunitySummaryBackendDict {
```

Comment:
I know the pattern of prefixing interface names with 'I' is widely used in the codebase, but I'd like to move away from this pattern for new code as it doesn't provide much value. Could you remove the 'I' prefix?

Reply:
Hmm should we open an issue to remove it? I don't think this PR should break away from the convention until we make a concrete plan to get rid of them all. @bansalnitish WDYT?

Reply:
As an aside -- @nishantwrp @srijanreddy98 @bansalnitish just to check, did one of you introduce this prefix? If so, I would like to understand why we did so and the reasoning behind it (this can be a separate email thread) so that we can figure out a convention for the whole codebase and make sure it's adhered to universally.

Reply:
I think the convention started here https://github.com/oppia/oppia/pull/9299#discussion_r429538466

Reply:
Looks like we've already filed #9348 to decide on a convention, but haven't yet.

Reply:
So, I guess this shouldn't block this PR.

Reply:
Per the thread -- from now on, please drop this prefix in PRs.